packages/gatsby-plugin-sharp/README.md

```diff
@@ -213,7 +212,7 @@ the original image, allowing the original image and its colors to partially
 "shine through". _Heads up_: If the original image contains an alpha channel it
 will be [flattened][15] before creating the composite.
 
-This works by adding an alpha channel to the duotone'd image - then we let Sharp
+This works by adding an alpha channel to the duotone'd image - then you let Sharp
```

Comment:
This is something the plugin does, so we would be appropriate.
```suggestion
This works by adding an alpha channel to the duotone'd image - then we let Sharp
```

Reply:
I'd recommend using "then the plugin lets Sharp do its magic"

Reply:
Thanks @laurieontech and @marcysutton. I will use Marcy's suggestion if that's cool?

Reply:
Really cool! 😎